sonar-project.properties

```diff
@@ -58,6 +58,7 @@ sonar.python.coverage.reportPaths= \
      deployment/coverage-reports/cli-coverage.xml, \
      deployment/coverage-reports/lambda-coverage.xml
 
+sonar.python.version=3.9
```

Comment:
why not include 3.10 and 3.11, like tox.ini?

Reply:
because the lambda only runs on 3.9. 3.10 and 3.11 testing is included only on the github action as a forward-looking due-dilligence